src/openrct2/peep/Peep.cpp

```diff
@@ -2655,7 +2655,7 @@ static void peep_go_to_ride_entrance(rct_peep * peep, Ride * ride)
 
     peep_decrement_num_riders(peep);
     peep->state     = PEEP_STATE_ENTERING_RIDE;
-    peep->sub_state = 1;
+    peep->sub_state = PEEP_RIDE_SS_IN_ENTRANCE;
```

Comment:
SS?

Reply:
Sub state. It was getting too ridiculously long the name